src/csharp/Grpc.Core.Tests/project.json

```diff
@@ -0,0 +1,35 @@
+{
```

Comment:
The NUnit test runner is nearly ready now, at which point in a dotnet-cli-only world we could ditch the entry point. But it makes sense to keep this for the moment - I can help get it more up to date when we want to.